client/ayon_maya/plugins/publish/collect_fbx_animation.py

```diff
@@ -34,3 +42,33 @@ def process(self, instance):
                 ))
             if skeleton_content:
                 instance.data["animated_skeleton"] = skeleton_content
+
+        attribute_values = self.get_attr_values_from_data(
+            instance.data
+        )
+
+        instance.data["upAxis"] = attribute_values.get(
+            "upAxis", self.up_axis)
+        instance.data["inputConnections"] = attribute_values.get(
+            "inputConnections", self.input_connections)
+
+    @classmethod
+    def get_attribute_defs(cls):
+        defs = [
+            UISeparatorDef("sep_fbx_options"),
+            UILabelDef("Fbx Options"),
+        ]
+        defs.extend(
+            super(CollectFbxAnimation, cls).get_attribute_defs() + [
+            EnumDef("upAxis",
+                    ["x", "y", "z"],
+                    default=cls.up_axis,
+                    tooltip="Define the up axis in your FBX file"),
```

Comment:
Does this influence how the geometry is written into the file, or is it just metadata? Like, does setting this _convert_ the scene's orientation. Would be good to have the description explain that?